modules/capi-yaml-machine-set-aws.adoc

```diff
@@ -16,28 +16,35 @@ kind: MachineSet
 metadata:
   name: <machine_set_name> # <1>
   namespace: openshift-cluster-api
+  labels:
+    cluster.x-k8s.io/cluster-name: <cluster_name> # <2>
 spec:
   clusterName: <cluster_name> # <2>
   replicas: 1
   selector:
     matchLabels:
       test: example
+      cluster.x-k8s.io/cluster-name: <cluster_name>
+      cluster.x-k8s.io/set-name: <machine_set_name>
```

Comment:
the label `cluster.x-k8s.io/set-name: <machine_set_name>` is machine label instead of machineset, here should be `machine.openshift.io/cluster-api-cluster: <cluster_name>`

Reply:
This is the machineSet selector for the machines it creates. It is correct here.

`here should be machine.openshift.io/cluster-api-cluster: <cluster_name>`
CAPI has different labels that serve the same purpose as the MAPI labels we currently have. 
During the MachineSet, Machine mirroring we will be converting between them. https://issues.redhat.com/browse/OCPCLOUD-2880